src/js/search/widgets/infinite_scroll.js

```diff
@@ -20,7 +20,11 @@ function renderTemplate(template, res) {
 }
 
 function scrolledNearBottom(el, offset) {
-  return (el.scrollHeight - el.scrollTop) < offset;
+  if(window.innerHeight < offset){
+    return (el.scrollHeight - el.scrollTop) < offset;
+  } else{
+    return (el.scrollHeight - el.scrollTop) < window.innerHeight* 2;
```

Comment:
Is there another way to do this? It's very magic-numbery and thus not hard to invision a situation where it doesn't work.